src/client.py

```diff
@@ -0,0 +1,27 @@
+import socket
+import sys
+
+
+infos = socket.getaddrinfo('127.0.0.1', 3000)
+
+stream_info = [i for i in infos if i[1] == socket.SOCK_STREAM][0]
+
+client = socket.socket(*stream_info[:3])
+
+client.connect(stream_info[-1])
+
+message = str(sys.argv[1:])
```

Comment:
This will put commas in between each of the words given on the command line. If you want to have spaces or some other character between words, use the string method `.join()`.